client/Rating&Reviews/ReviewsList/Photo.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class Photo extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      showModal: false
+    }
+
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick() {
+    const {showModal} = this.state;
```

Comment:
Very clean looking code. It took me a minute to realize that i could move the destructure statement to use methods!